documentation/modules/adding-source-provider.adoc

```diff
@@ -21,35 +21,140 @@ You can add {a-rhv} source provider by using the {ocp} web console.
+
+. Choose one of the following options for validating a CA certificate:
+
+** *Skip validation* : Migrate without validating a CA certificate.
+** *Use the system CA certiicate*: Migrate after validating the system CA certificate.
+** *Use a custom CA certificate*: Migrate after validating a custom CA certificate.
+
+.. To skip validation, select the *Skip certificate validation* check box.
+.. To validate the system CA certfiticate, leave the *Skip certificate validation* check box cleared and leave the *CA certificate* text box empty.
+.. To validate a custom CA certificate, leave the *Skip certificate validation* check box cleared and _either_ drag the CA certificate to the text box _or_ browse for it and click *Select*.
 endif::[]
 ifdef::rhv[]
-. Select *Red Hat Virtualization* from the *Provider type* list.
+. Click *Red Hat Virtualization*
+. Specify the following fields:
+thats
+* *Provider resource name*: Name to display in the list of providers
+* *URL*: API endpoint of the virtual machine being imported from {rhv-short}
+* *Username*: Username
+* *Password*: Password
+
+. Choose one of the following options for validating a CA certificate:
+
+** *Skip validation* : Migrate without validating a CA certificate.
+** *Use the system CA certiicate*: Migrate after validating the system CA certificate.
+** *Use a custom CA certificate*: Migrate after validating a custom CA certificate.
+
+.. To skip validation, select the *Skip certificate validation* check box.
+.. To validate the system CA certfiticate, leave the *Skip certificate validation* check box cleared and leave the *CA certificate* text box empty.
+.. To validate a custom CA certificate, leave the *Skip certificate validation* check box cleared and _either_ drag the CA certificate to the text box _or_ browse for it and click *Select*.
+endif::[]
+ifdef::ova[]
+. Click *Open Virtual Appliance (OVA)*.
 . Specify the following fields:
 
-* *Provider name*: Name to display in the list of providers
-* *RHV Manager host name or IP address*: {manager} host name or IP address -  if a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate
-* *RHV Manager user name*: {manager} user
-* *RHV Manager password*: {manager} password
+* *Provider resource name*: Name to display in the list of providers
+* *URL*: URL of the NFS file share that serves the OVA
+endif::[]
+ifdef::ostack[]
+. Click *Red Hat OpenStack Platform*.
+. Specify the following fields:
+
+* *Provider reasource name*: Name to display in the list of providers.
+* *URL*: {osp} Identity (Keystone) endpoint, for example, `http://controller:5000/v3`.
+* *Authentication type*: Choose one of the following methods of authentication and supply the information related to your choice. For example, if you choose *Application credential ID* as the authentication type, the *Application credential ID* and the *Application Secret* fields become active and you need to supply the ID and Secret.
+
```

Comment:
@mmartinv: These are the definitions I would like you to check. Any other comments are welcome, too, of course. Thanks.